### ISSUE TITLE
[vcpkg-ci-crashpad] CI build failure

### ISSUE THREAD

**Issue by talregev** [microsoft/vcpkg#40752]
### Operating system

Windows

### Compiler

msvc CI version

### Steps to reproduce the behavior

```Shell
run CI on
x86-windows
x64-windows
x64-windows-static
x64-windows-static-md
```

### Failure logs

https://dev.azure.com/vcpkg/public/_build/results?buildId=106590&view=logs&j=878666d5-db33-5b27-9e7d-b0c7ee352005&t=69176ca8-c93b-5311-0f74-6cb12a2b3f92&l=61492

**Comment by dg0yt** [comment]
The result of #40527. Analyzed two weeks ago in https://github.com/microsoft/vcpkg/pull/40519#issuecomment-2304229773.

**Comment by MonicaLiu0311** [comment]
Fixed by #40729.
